Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -391,6 +391,7 @@ WarpXOpenPMDPlot::WarpXOpenPMDPlot (
 {
     m_OpenPMDoptions = detail::getSeriesOptions(operator_type, operator_parameters,
                                                 engine_type, engine_parameters);
+    amrex::Print()<<".... openPMD options used: "<<m_OpenPMDoptions<<" \n";
```

Comment:
It would be ideal to format the output here as an "Info" message for consistency:
`amrex::Print() << Utils::TextMsg::Info(your_string) ;`

(note that you don't need to add `\n` at the end of the message)

Reply:
OK used Info to output  instead